.github/workflows/cd.yml

```
@@ -21,7 +21,7 @@ jobs:
       TF_VAR_db_passwd: ${{secrets.DB_PASSWORD}}
       TF_VAR_db_username:  ${{secrets.DB_USERNAME}}
       TF_VAR_github_token: ${{secrets.GITHUB_TOKEN_ORG}}
-      TF_VAR_hostname_default: ${{secrets.HOSTNAME_DEFAULT}}
+      TF_VAR_hostname_default: ${{secrets.HOSTNAME_API_DEFAULT}}
```

aqui ela tbm mudou de nome, agora ficou:
```suggestion
      TF_VAR_hostname_api_default: ${{secrets.HOSTNAME_API_DEFAULT}}
```